packages/desktop-ui-lib/src/lib/settings/plugins/component/plugin-marketplace/plugin-marketplace-item/plugin-marketplace-item.component.ts

```
@@ -0,0 +1,291 @@
+							name: this.plugin.source.name,
+							version: isUpdate
+								? this.plugin.versions[this.plugin.versions.length - 1]
+								: this.selectedVersion
+						},
+						registry: {
+							privateURL: this.plugin.source.registry,
+							authToken: this.plugin.source.authToken
+						}
+					},
+					contextType: 'npm'
+				});
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)